ably/ably-ui
# [MAW-509] Replace twitter logo with x logo on the footer

Jira Ticket Link / Motivation
----
https://ably.atlassian.net/browse/MAW-509



Summary of changes
----
Updated twitter logo with the new x logo



How do you manually test this?
----
Check footer component, x logo should appear instead of twitter


Reviewer Tasks (optional)
----




Merge/Deploy Checklist
----




Frontend Checklist
----
